src/libs/actions/Policy.js

```diff
@@ -43,25 +43,29 @@ function getSimplifiedEmployeeList(employeeList) {
  * Takes a full policy that is returned from the policyList and simplifies it so we are only storing
  * the pieces of data that we need to in Onyx
  *
- * @param {Object} fullPolicyOrPolicySummary
- * @param {String} fullPolicyOrPolicySummary.id
- * @param {String} fullPolicyOrPolicySummary.name
- * @param {String} fullPolicyOrPolicySummary.role
- * @param {String} fullPolicyOrPolicySummary.type
- * @param {String} fullPolicyOrPolicySummary.outputCurrency
- * @param {String} [fullPolicyOrPolicySummary.value.avatarURL]
- * @param {Object} [fullPolicyOrPolicySummary.value.employeeList]
+ * @param {Object}  fullPolicyOrPolicySummary
+ * @param {String}  fullPolicyOrPolicySummary.id
+ * @param {String}  fullPolicyOrPolicySummary.name
+ * @param {String}  fullPolicyOrPolicySummary.role
+ * @param {String}  fullPolicyOrPolicySummary.type
+ * @param {String}  fullPolicyOrPolicySummary.outputCurrency
+ * @param {String}  [fullPolicyOrPolicySummary.value.avatarURL]
+ * @param {Object}  [fullPolicyOrPolicySummary.value.employeeList]
+ * @param {Boolean} isSummary
  * @returns {Object}
  */
```

Comment:
We should also add `[fullPolicyOrPolicySummary.avatarURL]`, since that's where it lives in the policy summary object.